jerry-core/jerry.c

```diff
@@ -558,7 +558,7 @@ jerry_api_convert_eval_completion_to_retval (jerry_api_value_t *retval_p, /**< o
  */
 ssize_t
 jerry_api_string_to_char_buffer (const jerry_api_string_t *string_p, /**< string descriptor */
-                                 jerry_api_char_t *buffer_p, /**< output characters buffer */
+                                 jerry_api_char_t *buffer_p, /**< [out] characters buffer */
```

Comment:
It is arguable whether this "output" was ment to be a direction specifier or a name. The line below refers to this argument as the "output buffer". Alternatively, best of both worlds: `[out] output characters buffer`?